libs/components/forms/testing/src/checkbox/checkbox-group-harness.ts

```diff
@@ -86,6 +94,41 @@ export class SkyCheckboxGroupHarness extends SkyComponentHarness {
     return (await this.#getHeading()).hasClass('sky-screen-reader-only');
   }
 
+  /**
+   * The semantic heading level used for the checkbox group.
```

Comment:
can we keep the wording between the harness and development tab consistent?  my vote is to add the word `semantic` to the component docs